hudi-spark-datasource/hudi-spark/src/test/java/org/apache/hudi/table/TestHoodieMergeOnReadTable.java

```diff
@@ -541,7 +541,7 @@ public void testLogBlocksCountsAfterLogCompaction(boolean populateMetaFields, St
   @ParameterizedTest
   @ValueSource(booleans = {true, false})
   public void testMetadataStatsOnCommit(Boolean rollbackUsingMarkers) throws Exception {
-    HoodieWriteConfig cfg = getConfigBuilder(false, rollbackUsingMarkers, IndexType.INMEMORY)
+    HoodieWriteConfig cfg = getConfigBuilder(false, rollbackUsingMarkers, IndexType.BLOOM)
```

Comment:
Reason of changing to `BLOOM`?

Reply:
Removed it. This was part of HUDI-1517. The test passes even without the change.